qp_klp/tests/test_step.py

```diff
@@ -419,6 +426,68 @@ def _create_test_input(self, stage):
             with open(fake_path, 'w') as f:
                 f.write("This is a file")
 
+        if stage >= 5:
```

Comment:
why 5?

Reply:
The 5 was so that it wouldn't interfere with legacy tests. Looking at it again there's something I want to tweak. I'll keep you posted.